src/_includes/docs/add-to-app/ios-project/limits-common-deps.md

```diff
@@ -0,0 +1,18 @@
+
+Flutter can't handle [common dependencies with xcframeworks][common].
+If both the host app and the Flutter module's plugin define the
+same pod dependency and you integrate Flutter module using this option,
+errors result.
+These errors include issues like Multiple commands produce
+'CommonDependency.framework'.
+
+To work around this issue, link every plugin source in its `podspec` file
+from the Flutter module to the host app's `Podfile`.
+Do this instead of using the plugins' `xcframework` files that the
```

Comment:
FYI, the `Foo.xcframework` is actually a directory. For all intents and purposes you can act like it's a file, but I would avoid language like "`xcframework` files".